### DISCUSSION TITLE
在new Graph()启用了调整大小，能不能设置单独的几个节点不允许调整大小。

### DISCUSSION THREAD

**Discussion by anyone0034** [antvis/X6#2592] [Q&A]
1. 现在是全部所有的节点，都可以调整大小，我有1个应用场景，有些节点，不允许调整大小。

![微信截图_20220830141130](https://user-images.githubusercontent.com/15609924/187365092-5a9aaffc-979c-423d-bf07-6d742c32eddf.png)

**Comment by zmcode** [comment]
https://x6.antv.vision/zh/docs/api/graph/transform#resizing
![image](https://user-images.githubusercontent.com/48686959/189805605-dddcec95-f77c-4947-910b-de01e459c879.png)

**Comment by ghost** [comment]
2.x中怎么做
